### ISSUE TITLE
print Canvas to printer

### ISSUE THREAD

**Issue by akira32chen** [sanketrandive/Unity-Printer-Function#1]
Can this plugin print  Canvas to printer?

**Comment by sanketrandive** [comment]
Yes it can possible, but not with this plugin i am developing new plugin of
that part though. I will publish the new package as soon as possible within
this or next week.

**Comment by sanketrandive** [comment]
Hey hi,
I have added canvas and camera print in this. Please have look and test it. Please do let me know if their is any issue.
